src/main/java/com/notably/view/BlockContentViewImpl.java

```diff
@@ -0,0 +1,40 @@
+package com.notably.view;
+
+import javafx.fxml.FXML;
+import javafx.scene.control.TextArea;
+import javafx.scene.layout.Region;
+
+/**
+ *
+ */
+public class BlockContentViewImpl extends ViewPart<Region> {
```

Comment:
Hmmm, we don't really need to call it `BlockContentViewImpl`? We can just call it `BlockContentView` directly

Reply:
But, perhaps, you would want to call this `BlockContentEditView` (the text area view)? I think we need to differentiate it from `BlockContentDisplayView` (just a display) somehow. Maybe we can make this change in a future PR?

Reply:
For now, I'll change it to BlockContentEditView! If a future PR needs the distinction I will do it then :)